soc/arm/st_stm32/stm32h7/Kconfig.soc

```diff
@@ -77,4 +77,8 @@ config SOC_STM32H7B3XXQ
 	select CPU_CORTEX_M7
 	select CPU_HAS_FPU_DOUBLE_PRECISION
 
+config STM32H7_DUAL_CORE
+	bool "Dual Core"
+	default y
+
```

Comment:
It should be moved outside of choice structure.

Reply:
ok